roles/common/tasks/main.yml

```diff
@@ -53,6 +54,7 @@
 - include: ssh.yml
 
 - include: networking.yml
+  tags: ['prereboot']
```

Comment:
If we're going to tag a whole include file we should comment the file so that later devs are aware of the implications of that file, that it could be ran from master at any time on any cluster.